packages/docs/src/pages/search.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { TextField } from '@cmsgov/design-system';
+import { useFlexSearch } from 'react-use-flexsearch';
+import { useStaticQuery, graphql } from 'gatsby';
+import { MdxQuery, SearchDataStore, SearchQuery } from '../helpers/graphQLTypes';
+import Layout from '../components/layout/Layout';
+import useTheme from '../helpers/useTheme';
+
+const stripHTML = (s) => {
+  if (s === null || s === '') {
+    return false;
+  } else {
+    s = s.toString();
+    s = s.replace(/#/gi, '');
+    return s.replace(/(<([^>]+)>)/gi, '');
+  }
+};
+
+const SearchPage = ({ location }: MdxQuery) => {
+  const [query, setQuery] = useState('');
+  const theme = useTheme();
+
+  useEffect(() => {
+    const params = new URLSearchParams(location.search);
+    setQuery(params.get('query'));
+  }, [location.search]);
+
+  const searchData: SearchQuery = useStaticQuery(graphql`
+    {
+      localSearchPages {
+        index
+        store
+      }
+    }
+  `);
+
+  const { index, store } = searchData.localSearchPages;
+  const results = useFlexSearch(query, index, store);
+
+  return (
+    <Layout
+      frontmatter={{
+        title: 'Search this site',
+      }}
+      location={location}
+      slug="Search"
+      theme={theme}
+    >
+      <div id="search-results">
+        <div className="search-form ds-u-margin-bottom--4">
+          <TextField
```

Comment:
This might need to have the `search` type on it since it's acting as a search field.